### PR TITLE
Added gatsby to the project

### LINKED ISSUE #61: Setup Gatsby in the monorepo

### PR DESCRIPTION
**Pull request by sonalranjit**
- Added gatsby to replace the frontend create-react-app. 
- Still requires to update the docs to reflect these changes.

### CODE REVIEW

File: .github/workflows/main.yml
Comment (christikaes) [inline .github/workflows/main.yml:27]: I think it's fine to call this client still - because it's our client app.

File: packages/unitednepali-gatsby/README.md
Comment (christikaes) [inline packages/unitednepali-gatsby/README.md:10]: 🤔 what were you thinking here?

**Review by christikaes — COMMENTED** [review]
Looks great! Added a small comment, and will look again once the readme is updated.

Also, we'll need to update the netlify commands. Is there a better way of doing netlify deploys? Right now we just have a script in netlify - is there a way to add a netlify config file (similar to deploying to firebase)?

File: packages/unitednepali-gatsby/README.md
Comment (sonalranjit) [reply packages/unitednepali-gatsby/README.md:10]: lol this was added by lerna i believe

Comment (christikaes) [reply packages/unitednepali-gatsby/README.md:10]: 😂

**Comment by sonalranjit** [comment]
@christikaes should be good to go now, need a second review

**Review by christikaes — APPROVED** [review]
Looks great! Thanks for fixing the deploy too :D

**Pull request merged**
